### PR TITLE
enable plugins in integration tests

### PR DESCRIPTION
**Pull request by Thanhphan1147**
Enable all plugins when running integration tests using the `rake site_settings:import` task

```
echo '{settings}' | /srv/discourse/app/bin/bundle exec rake site_settings:import -
```

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Manging Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated.
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

**Review by javierdelapuente — DISMISSED** [review]

**Review by merkata — DISMISSED** [review]
Nice!!

File: tests/integration/conftest.py [hunk 1]
```diff
@@ -193,6 +201,14 @@ async def app_fixture(
...
     )
     await model.wait_for_idle(status="active", raise_on_error=False)
+    inline_yaml = "\n".join(f"{plugin}_enabled: true" for plugin in ENABLED_PLUGINS)
+    enable_plugins_command = (
+        "pebble exec --user=_daemon_ --context=discourse -w=/srv/discourse/app -ti -- /bin/bash -c "
+        f'''"echo '{inline_yaml}' | /srv/discourse/app/bin/bundle exec rake site_settings:import -"'''
+    )
+
+    action = await unit.run(f"/bin/bash -c '{enable_plugins_command}'")
+    logger.info(action.results)
```
Comment (yanksyoon) [inline tests/integration/conftest.py:211]: Out of curiosity, would this action be optional and not need to be awaited?

**Review by yanksyoon — DISMISSED** [review]
LGTM!

File: tests/integration/conftest.py (hunk 1 above)
Comment (Thanhphan1147) [reply tests/integration/conftest.py:211]: I think we should await the completion of this action as we want to test that discourse can function properly with all of the plugins enabled.

Comment (yanksyoon) [reply tests/integration/conftest.py:213]: Oh sorry for the vague statement, let me try to clarify :D
```
action = await unit.run(block=True) # note the block=True here or
await action.wait() # to wait for the results
assert action.results.["property_here"] == "Expected results"
```

From the juju docs of the action:
> blocl: Useful with older versions of juju, i.e. getting the result without having to call action.wait() separately.
:returns: A juju.action.Action instance.
Note that this is very similarly to unit.run_action only enqueues the action. You will need to call action.wait() on the resulting Action instance if you wish to block until the action is complete.

**Review by yanksyoon — DISMISSED** [review]

**Review by javierdelapuente — APPROVED** [review]

**Review by merkata — APPROVED** [review]

**Pull request merged**